examples/bakdata/atm-fraud-detection/defaults.yaml

```diff
@@ -3,6 +3,9 @@ pipeline-component:
 
 kubernetes-app:
   namespace: ${NAMESPACE}
+  app:
+    image_tag: "1.0.0"
+    replica_count: 1
```

Comment:
```suggestion
    imageTag: "1.0.0"
    replicaCount: 1
```

Reply:
why do you want me to revert it?

Reply:
For the examples, I would stick to only one naming convention. Since there is camelCase here already I would stick to camel case.

Reply:
ah ok, there are only two occurrences of camel case left in this example. would it be ok if I change these to snake case as well? 😄 or does it have to be camel case?

Reply:
I am totally fine with the snake case but also pay attention to the other files here:
https://github.com/bakdata/kpops/blob/0b882c7acddacfe2ce037bf7563183d5af3ead28/examples/bakdata/atm-fraud-detection/pipeline.yaml#L42

Reply:
that's an annotation, they don't adhere to any convention 🤷🏻‍♂️

Reply:
that's why we can't convert it

Reply:
ah damn 😭 didn't see that.